packages/hash/frontend/src/pages/[account-slug]/entities/[entity-id].page/entity-editor/properties-section/property-table.tsx

```diff
@@ -65,6 +65,7 @@ export const PropertyTable = ({
         showSearch={showSearch}
         onSearchClose={onSearchClose}
         customRenderers={customRenderers}
+        resizable
```

Comment:
Should we be passing this for links table too?

Reply:
I think we'll consider this when we advance more on links table. Sounds reasonable.